ext/date/php_date.c

```diff
@@ -1154,7 +1154,7 @@ static zend_string *date_format(char *format, size_t format_len, timelib_time *t
 
 			/* year */
 			case 'L': length = slprintf(buffer, 32, "%d", timelib_is_leap((int) t->y)); break;
-			case 'y': length = slprintf(buffer, 32, "%02d", (int) t->y % 100); break;
+			case 'y': length = slprintf(buffer, 32, "%02lld", (zend_long) t->y % 100); break;
```

Comment:
Printing a 2-digit number as long long doesn't make sense. Maybe you're looking to change the precedence of the cast? `(int) (t->y % 100)`.

Reply:
You're absolutely right, will do that instead